### DISCUSSION TITLE
proxyState is not iterable

### DISCUSSION THREAD

**Discussion by MichaelHao25** [pmndrs/valtio#774] [Bug report]
## Summary

refer to :  https://github.com/pmndrs/valtio/issues/705

Now I am developing an npm package, and I use valtio, so users who use my package cannot use valtio. How to solve this problem. It can be solved by referring to the above address, but I hope the official will give a better solution, thank you

## Check List

Please do not ask questions in issues.

- [x] I've already opened a [discussion](https://github.com/pmndrs/valtio/discussions) before opening this issue, or already discussed in other media.

Please include a minimal reproduction.

- [x] I've added a link to a [typescript playground](https://www.typescriptlang.org/play) or [codesandbox](https://codesandbox.io) with a minimal reproduction.

**Comment by dai-shi** [comment]
I think the official solution is to use `valtio` as "peerDependencies".

**Reply by MichaelHao25** [reply]
Thank you very much！！！
